server/WebAPI/Services/Assets/AssetType.cs

```diff
@@ -2,6 +2,7 @@
 {
   public enum AssetType
   {
-    UserAvatar = 1
+    UserAvatar = 1,
+    Entertainment = 3
```

Comment:
where 2?

Reply:
Because Irina uses 2 in her pull request

Reply:
and what type is it?